website/app/controllers/application.js

```diff
@@ -100,10 +100,12 @@ export default class ApplicationController extends Controller {
     // Disable transition validation if we're transitioning to the same page
```

Comment:
[suggestion] since the logic has changed, maybe we could also update the comment above it, to reflect the logic below